src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -66,7 +66,7 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
 
         <div class="col-md-4 col-sm-5 col-xs-5 wi-state-btn pull-right padding0">
           <div dropdown class="dropdown card-pf-time-frame-filter detail-status-dropdown">
-            <button dropdown-open 
+            <button dropdown-open [disabled]='addNewWI'
```

Comment:
Why do you need to keep it disabled until the item is added? I understand that changing it makes an API call for update but you can use the same logic in the event listener. In fact you can have all the fields like Description, States, Assignee, just don't let any of the vent listener call the save method until you have workItem.attributes['system.title']